### DISCUSSION TITLE
Example using C API on Linux environment

### DISCUSSION THREAD

**Discussion by JAEWOOKe** [microsoft/onnxruntime#6904] [Other Q&A]
Dear all, 

I have cross-compiled for ARM from Ubuntu and checked **onnxruntime_test_all** test passed on the ARM board.

However, I am quite lost on how to use **libonnxruntime.so** through C API to run inference sessions like running on Python.

I am following the [document](https://www.onnxruntime.ai/docs/reference/api/c-api.html) but not sure how to run it.

Can anyone show me how it should be done?

Thanks,
Jaewook

**Comment by snnn** [comment]
https://github.com/microsoft/onnxruntime/blob/master/samples/c_cxx/fns_candy_style_transfer/fns_candy_style_transfer.c
